HardwareTests/src/mod/cli/cli.c

```diff
@@ -60,6 +60,7 @@ int cmdsProcessed = 0;
 // ----------------------
 void processLine();
 void CliShowStatistics(int argc, char *argv[]);
+void MyOwnFunction(void);
```

Comment:
You did it right here but this should go to Your module (The LED module in this case).